docs/_guide/styles.md

```diff
@@ -220,9 +220,30 @@ There are some important caveats though:
 * External styles can cause a flash-of-unstyled-content (FOUC) while they load.
 * The URL in the `href` attribute is relative to the _main document_, making this technique mostly useful for application elements where asset URLs are well known, and not for reusable elements published publicly.
 
+Now, be sure what are you doing before do this: by the way, if you `really need` or ` really want` a `separate css file from main component file`, you can use the `unsafeCSS` function to import the "external css" like the example below.
+
+```js
+import {LitElement, css, html} from 'lit-element';
+import style from './style.css';
```

Comment:
This will return a CSSStyleSheet, so passing it to `unsafeCSS` will not work.

Reply:
I'm using this with .stylus, but work with sass too, I can commit this example with a .sass, what you say ?

Reply:
This doesn't work at all in the platform yet, and when it does work it'll return a CSSStyleSheet object, see http://w3c.github.io/webcomponents/proposals/css-modules-v1-explainer

I would rather not document importing CSS at all like this until it's available in browsers.